jdaviz/configs/specviz/tests/test_helper.py

```diff
@@ -248,7 +250,9 @@ def test_load_spectrum_list_directory(tmpdir, specviz_helper):
 
     with pytest.warns(UserWarning, match='SRCTYPE is missing or UNKNOWN in JWST x1d loader'):
         specviz_helper.load_spectrum(data_path)
-    assert len(specviz_helper.app.data_collection) == 3
+    # NOTE: the length was 3 before specutils 1.9 (https://github.com/astropy/specutils/pull/982)
+    expected_len = 40 if Version(specutils.__version__) >= Version('1.9.0') else 3
```

Comment:
This would make it easier to grep in the future when we bump minversion but it also depends on how devs grep for it. This is just something we do over at `astropy`, so I'll leave it at your discretion.

```suggestion
    SPECUTILS_LT_1_9 = Version(specutils.__version__) < Version('1.9.0')
    expected_len = 40 if not SPECUTILS_LT_1_9 else 3
```